src/AEG/AEGTree.ts

```diff
@@ -29,24 +28,24 @@ export class AEGTree {
      * @returns True, if the structure is structurally consistent. Else, false.
      */
     private verifyAEG(currentCut: CutNode): boolean {
-        for (let i = 0; i < currentCut.children.length; i++) {
+        for (let i = 0; i < currentCut.Children.length; i++) {
```

Comment:
I'm not a fan of the capitalized setter and getter names, should be identical in case to the underlying member variable name (I mention this later in another file as well).